### PR TITLE
Script to populate db with recipe ids

### PR DESCRIPTION
**Pull request by milessorce**
Look closely at the part under "forEach"

### CODE REVIEW

**Comment by irbab00n** [comment]
Approved

File: server/populateIdDatabaseScript.js [hunk 1]
```diff
@@ -28,20 +28,36 @@ const foodTypes = [
...
-};
- 
-rp(options)
+foodTypes.forEach(foodType => {
+  rp({
+    uri: `http://api.yummly.com/v1/api/recipes?_app_id=${config.APP_ID}&_app_key=${config.APP_KEY}&allowedDiet[]=392^Vegetarian&q=${foodType}&requirePictures=true&maxResult=${resultLimit}`
+  })
   .then(data => {
-    console.log(`Data was succesfully searched fetched`, data);
+    data = JSON.parse(data);
```
Comment (jmaxc6) [inline server/populateIdDatabaseScript.js:41]: Are we sure the data is going to be returned as a JSON stringified object?

File: server/populateIdDatabaseScript.js [hunk 2]
```diff
@@ -28,20 +28,36 @@ const foodTypes = [
...
 
-
-var options = {
-  uri: `http://api.yummly.com/v1/api/recipes?_app_id=${config.APP_ID}&_app_key=${config.APP_KEY}&allowedDiet[]=392^Vegetarian&q=${foodParameter}&requirePictures=true&maxResult=${resultLimit}`,
-  json: true
-};
- 
-rp(options)
+foodTypes.forEach(foodType => {
+  rp({
```
Comment (jmaxc6) [inline server/populateIdDatabaseScript.js:37]: How is the API being queried with this command (it just uses 'rp'; I don't see a specific method being invoked)?

File: server/populateIdDatabaseScript.js [hunk 3]
```diff
@@ -28,20 +28,36 @@ const foodTypes = [
...
+    uri: `http://api.yummly.com/v1/api/recipes?_app_id=${config.APP_ID}&_app_key=${config.APP_KEY}&allowedDiet[]=392^Vegetarian&q=${foodType}&requirePictures=true&maxResult=${resultLimit}`
+  })
   .then(data => {
-    console.log(`Data was succesfully searched fetched`, data);
+    data = JSON.parse(data);
+    data.forEach(recipe => {
+      if (recipe.rating >= 4) {
+        let newRecipe = new Recipe(data);
```
Comment (jmaxc6) [inline server/populateIdDatabaseScript.js:44]: Shouldn't this be 'let newRecipe = new Recipe(recipe)'?

File: server/populateIdDatabaseScript.js (hunk 2 above)
Comment (jmaxc6) [reply server/populateIdDatabaseScript.js:37]: This is fine. Rp makes sense

File: server/populateIdDatabaseScript.js (hunk 1 above)
Comment (jmaxc6) [reply server/populateIdDatabaseScript.js:41]: Yes-confirmed by Myles

File: server/populateIdDatabaseScript.js (hunk 3 above)
Comment (jmaxc6) [reply server/populateIdDatabaseScript.js:44]: Yes-we're going to make the change now

**Pull request merged**